### PR TITLE
[12.0][FIX] pos_payment_method_cashdro: floating point quirks

### PR DESCRIPTION
**Pull request by chienandalu**
Overcome floating point issues: https://stackoverflow.com/a/588014

For expample 2.3 * 100 ->  229.99999999999997

cc @Tecnativa TT39975

please review @CarlosRoca13

### CODE REVIEW

File: pos_payment_method_cashdro/static/src/js/screens.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ odoo.define("pos_payment_method_cashdro.screens", function (require) {
             // Cashdro treats decimals as positions in an integer we also have
             // to deal with floating point computing to avoid decimals at the
             // end or the drawer will reject our request.
-            var amount = parseInt(order.get_due(payment_line).toFixed(2) * 100);
+            var amount = Math.round(order.get_due(payment_line) * 100);
```
Comment (pedrobaeza) [inline pos_payment_method_cashdro/static/src/js/screens.js:71]: No 2 decimals indication? Or is it the default?

Comment (chienandalu) [reply pos_payment_method_cashdro/static/src/js/screens.js:71]: It doesn't have the same meaning as python `round()`.  From mozdev (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/round):

> The `Math.round()` function returns the value of a number **rounded to the nearest integer**.

So for a price of **2.30**, the Cashdro expects an input of **230**, but in JS: **2.3 * 100 = 229.99999999999997** (you can test it in your browser :slightly_smiling_face: ) due to floating point operations accuracy.

Comment (pedrobaeza) [reply pos_payment_method_cashdro/static/src/js/screens.js:71]: OK then.

**Review by pedrobaeza — APPROVED** [review]

**Review by CarlosRoca13 — APPROVED** [review]

**Pull request merged**